Formula/x3270.rb

```diff
@@ -10,17 +10,18 @@ class X3270 < Formula
     sha256 "897d18de2b85172e751719687710f20b1717293b10dac669d6eb052e08d4b02a" => :yosemite
   end
 
+  option "without-x11", "Exclude x3270 (X11-based version)"
```

Comment:
Probably wants flipped to `with-x11` now, sorry, should have noted that last time.